addons/binding/org.openhab.binding.te923/src/main/java/org/openhab/binding/te923/internal/ExecUtil.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.te923.internal;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Wrapper around exec functions. Merge error and command line result
+ *
+ * @author Gabriel Klein - Initial contribution
+ */
+public class ExecUtil {
+
+    /**
+     * Execute some command
+     *
+     * @param string
+     * @return
+     */
+    public static String exec(String cmd) {
+
+        Logger logger = LoggerFactory.getLogger(ExecUtil.class);
+        String res;
+
+        synchronized (ExecUtil.class) {
+
+            int COMMAND_TIMEOUT = 10000;
+
+            Process proc = null;
+            try {
+                Runtime rt = Runtime.getRuntime();
+                proc = rt.exec(cmd);
+            } catch (Exception e) {
+                logger.error("An exception occurred while executing '{}' : '{}'", new Object[] { cmd, e.getMessage() });
```

Comment:
Only log to error if it's a nonrecoverable system bug. also no need to `new Object`.